src/postgres/schema.sql

```diff
@@ -0,0 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE bytecode (
+    id uuid PRIMARY KEY,
+    address text NOT NULL,
+    creation_bytecode text NOT NULL,
+    chain_id integer NOT NULL,
+    block_number integer NOT NULL,
```

Comment:
Ah if we add the block number (which is useful information) we have to check to select the largest one for querying the current deployed bytecode hash.

Reply:
Ah sorry, there is another primary key anyway.

Reply:
We talked about storing all the changes from oldest to latest, but to show users only the latest change. In the future if we want to track changes in time we could easily support this and you could track changed for a contract in time.